docs/testing.rst

```diff
@@ -167,19 +170,191 @@ the relevant place and then create a new ``test_FILENAMEHERE.js`` file
+   2. run ``./manage.py celeryd`` to launch celery server
+   3. in another terminal, run ``./manage.py runserver``
+   4. open up a browser
+   5. log in to the server
+   6. go to admin
+   7. go to *Elasticssearch maintenance*
+   8. launch a reindexing
+
+      Make sure it's reindexing things. Once you know it's reindexing
+      things, then you can cut it short. Otherwise it takes *forever.
+
+
+Testing localizations
+---------------------
+
+1. make sure your ``locale/`` directory is up to date
+2. run ``./manage.py extract`` and make sure that it produced or updated
+   a ``locale/templates/LC_MESSAGES/django.pot`` file and that msgids
+   did not change
+3. run ``./manage.py merge`` and make sure it did the right thing
+4. run ``./bin/compile-linted-mo.sh`` and make sure dennis linted the
+   ``.po`` files and that the script compiled ``.mo`` files
+5. verify that localizations work:
+
+   1. run ``./manage.py runserver`` to launch the server
+   2. open up a browser
+   3. go to the front page dashboard and look at it in French and make
+      sure all strings are translated
+   4. leave feedback and make sure feedback form is in French
+
+
+Testing analyzer section
+------------------------
+
+Verify analyzer views load:
+
+1. run ``./manage.py runserver`` to launch the server
+2. open up a browser
+3. log in to the server
+4. go to analyzer section
+5. make sure all the views load
+
+
+Testing admin
+-------------
+
+Verify the admin views load:
+
+1. run ``./manage.py runserver`` to launch the server
+2. open up a browser
+3. log in to the server
+4. go to admin
+5. make sure all the admin views load
+
+
+Testing cron jobs
+-----------------
+
+1. for each job in ``bin/crontab/crontab.tpl``, make sure it works
+
+
+Testing documentation
+---------------------
+
+Verify that the documentatino builds:
+
+1. run ``cd docs/``
+2. run ``make html`` and make sure there are no build errors
+
+
+Testing vagrant
+---------------
+
+With an existing vagrant environment:
+
+1. run ``vagrant up``
+2. run ``./peep.sh install -r requirements/requirements.txt``
+3. run ``./peep.sh install -r requirements.dev.txt``
+4. run ``vagrant ssh``
 
-Unless the current behavior, and thus the test that verifies that
-behavior is correct, is demonstrably wrong, don't change tests. Tests
-may be refactored as long as its clear that the result is the same.
+   1. run ``cd fjord``
+   2. run ``./py.test``
 
+Now we're going to create a new vagrant environment:
 
-Removing tests
-==============
+1. run ``vagrant halt``
+2. run ``vagrant destroy --force``
+3. run ``vagrant up``
+4. run ``vagrant ssh``
 
-On those rare, wonderful occasions when we get to remove code, we
-should remove the tests for it, as well.
+   1. run ``cd fjord``
+   2. run ``./py.test``
 
-If we liberate some functionality into a new package, the tests for
-that functionality should move to that package, too.
+If that works, then it probably works fine in vagrant development
```

Comment:
> If that works, then it probably works fine in vagrant development environment.

I don't understand this sentence. Do you mean that if the tests work in the Vagrant environment, it is likely that everything else does too?